system-addon/lib/ActivityStream.jsm

```diff
@@ -26,6 +26,7 @@ const {FaviconFeed} = ChromeUtils.import("resource://activity-stream/lib/Favicon
 const {TopSitesFeed} = ChromeUtils.import("resource://activity-stream/lib/TopSitesFeed.jsm", {});
 const {TopStoriesFeed} = ChromeUtils.import("resource://activity-stream/lib/TopStoriesFeed.jsm", {});
 const {HighlightsFeed} = ChromeUtils.import("resource://activity-stream/lib/HighlightsFeed.jsm", {});
+const {MessageCenterFeed} = ChromeUtils.import("resource://activity-stream/lib/MessageCenterRouter.jsm", {});
```

Comment:
nit: File name and import name don't match which makes it kinda confusing?

Reply:
fair enough, I'll move to a different file